### ISSUE TITLE
Search: is:poll

### ISSUE THREAD

**Issue by jtagcat** [zedeus/nitter#216]
There's currently no way to filter for polls (of that I know of).

**Comment by zedeus** [comment]
There is a way to filter polls but it would require more checkboxes which is counter productive to #217

**Comment by jtagcat** [comment]
Well the result would be to have twice the functionality in current space.

#217 is about 'this feature takes way more space than it needs for the same amount of functionality, let's make this more space efficient (and actually speedier to use)'

**Comment by zedeus** [comment]
It could be possible using buttons with three states (neutral, filter, exclude) but I think that requires JavaScript.

**Comment by jtagcat** [comment]
I still can't see why you would need neutral (all neutrals could be done by (dis)select all. Could you provide a few examples?

**Comment by zedeus** [comment]
Neutral would be the same as not selected, but then you still need a way to represent the filter and exclude states.

**Comment by B0pol** [comment]
> but I think that requires JavaScript.

I don't think so, look at https://html.duckduckgo.com/html/?q=test; you can choose region and time without JavaScript, it will just send a new request, but that's exactly how filters works today.

For example we could have `&nativeretweets=filter` or just `&nativeretweets=f` in the url parameter.

**Comment by zedeus** [comment]
I don't see how this is related to the issue, `&f-nativeretweets=on` is how it works today (=f/=e is not possible with checkboxes). The duckduckgo example is just a dropdown menu, not a custom checkbox with three states.

**Comment by zedeus** [comment]
This can be done like so: `card_name:poll2choice_text_only OR card_name:poll2choice_image OR card_name:poll2choice_video OR card_name:poll3choice_text_only OR card_name:poll3choice_image OR card_name:poll3choice_video OR card_name:poll4choice_text_only OR card_name:poll4choice_image OR card_name:poll4choice_video`

I've also found a way to search for gifs, so I'll be replacing two of the current filters with that soon (probably "Pro videos" and "Safe").
